File: apps/android/app/src/main/java/com/michaeldadi/sprout/config/CognitoConfig.kt
package com.michaeldadi.sprout.config

/**
 * Cognito configuration object
 * Mirrors the iOS CognitoConfig.swift structure
 */
object CognitoConfig {
    // These values will be populated from CDK outputs
    val region = AppConfig.awsRegion
    val userPoolId = AppConfig.cognitoUserPoolId
    val clientId = AppConfig.cognitoClientId
}

File: apps/ios/Sprout/Services/CognitoConfig.swift
import Foundation

struct CognitoConfig {
    static let shared = CognitoConfig()
    
    // These values will be populated from CDK outputs
    let region = AppConfig.shared.awsRegion
    let userPoolId = AppConfig.shared.cognitoUserPoolId
    let clientId = AppConfig.shared.cognitoClientId
    
    private init() {}
}
